docs/docs/mdx/importing-and-using-components.md

```diff
@@ -117,4 +117,4 @@ Here is a demo:
 
 ### Additional resources
 
-- Follow this detailed [example on using MDX](/examples/using-MDX) to import and render components.
+- Follow this detailed [example on using MDX](https://github.com/gatsbyjs/gatsby/tree/master/examples/using-MDX) to import and render components.
```

Comment:
I think we prefer relative links as per https://www.gatsbyjs.org/contributing/gatsby-style-guide/#use-relative-hyperlinks-for-local-links

Reply:
Oh, but this is not on the site, right. Never mind.